netmaster/main.go

```diff
@@ -209,6 +212,9 @@ func (d *daemon) ListenAndServe() {
 		})
 	}
 
+	// proxy Handler
+	router.PathPrefix("/proxy/").HandlerFunc(proxyHandler)
```

Comment:
I am not very clear on the use case for having proxy embedded inside netmaster. 

Shouldn't it be provided as a separate service like kube-proxy in kubernetes cluster? This will allow us to use other proxies later like nginx, ha-proxy etc, right?